protocol.md

```diff
@@ -2,18 +2,18 @@
 
 The following terms are used in this document:
 
-* **server** : A dqlite node.
-* **client** : Either application code (typically wanting to issue database queries) or a dqlite node (typically pushing replicational data).
-* **connection** : A TCP or Unix socket connection established by the client against a server.
-* **word** : A sequence of 8 bytes.
-* **protocol version** : A positive number stored in a word using little endian representation.
-* **message** : A sequence of bytes sent either by the client to the server or by the server to the client. It consists of a header and a body. The header consists of a single word with the following layout:
+* *server* : A Dqlite node.
+* *client* : Either application code (typically wanting to issue database queries) or a Dqlite node (typically pushing replicational data).
+* *connection* : A TCP or Unix socket connection established by the client against a server.
+* *word* : A sequence of 8 bytes.
+* *protocol version* : A positive number stored in a word using little endian representation.
+* *message* : A sequence of bytes sent either by the client to the server or by the server to the client. It consists of a header and a body. The header consists of a single word with the following layout:
   * byte 0 to 3: Size of the message body, expressed in words and stored using little endian representation. For example a value of [2 1 0 0] means that the message body consists of 258 bytes.
-  * byte 4: Type code identifiying the schema of the message body.
+  * byte 4: Type code identifying the schema of the message body.
   * byte 5: Message schema version. This version is bumped for a particular message type when the expected format of the body for that message type changes. Unless otherwise noted, the message formats described in this document are for schema version 0.
   * byte 6 to 7: Currently unused.
 
-The message body is a sequence of fields as described by the associated schema. Message types and their schemas are listed below.
+The message body is a sequence of fields as described by the associated schema. Message types and their schemata are listed below.
```

Comment:
schemata feels a bit archaic?

Reply:
OK, I guess aspell is a bit archaic - the current dictionaries actually allow both. :)
Changing back.